cmd/relay/relay/ingest.go

```diff
@@ -159,6 +160,7 @@ func (r *Relay) processSyncEvent(ctx context.Context, evt *comatproto.SyncSubscr
 
 	if !acc.IsActive() {
 		logger.Info("dropping sync message for non-active account", "status", acc.Status, "upstreamStatus", acc.UpstreamStatus)
+		EventsWarningsCounter.WithLabelValues(hostname, "non-active").Add(1)
```

Comment:
I think the "warn" label value should be "inactive-account", not "non-active" (the log line above could also be changed from "non-active" to "inactive").

This also applies to line 103 in the same file.